package.json

```diff
@@ -3,8 +3,8 @@
   "version": "0.0.1",
   "description": "A set of scripts to play the BitBurner game",
   "scripts": {
-    "build": "babel ./src/* -d dist",
-    "server": "node ./devScripts/webserver.js -p 8080 -d ./src --enable_cors --verbose",
+    "build": "rollup -c ./rollup.config.js",
```

Comment:
I think babel is still required to get the tests working although it's no longer needed in the "build" step. That's why it has been removed here but not removed as a dev dependency.